src/Stateless/StateConfiguration.Async.cs

```diff
@@ -236,7 +236,7 @@ public StateConfiguration OnEntryAsync(Func<Task> entryAction, string entryActio
                 Enforce.ArgumentNotNull(entryAction, nameof(entryAction));
                 return OnEntryAsync(
                     t => entryAction(),
-                    entryActionDescription ?? entryAction.TryGetMethodName());
+                    entryActionDescription);
```

Comment:
We're still losing the method name in these calls (I think the change applied to the sync class is needed here, too).